docs/data-management/qaqc-criteria-unprocessed.md

```diff
@@ -57,7 +57,10 @@ The following lists the pre-defined exclusion criteria for analyses of whole-bra
 
 ## Task behavior
 
-For accurate estimation of task activation, it is essential to check the quality of both the fMRI images and the task behavior [Etzel 2023]. As such, it is important to verify that the subjects were attempting to perform the instructed task (i.e not sleeping and not responding randomly). One particularity of this project is that our single participant is the principal investigator of the project. As such, we trust that the participant, when actively awake, has enough incentive to follow the tasks accurately, even more so that the tasks are simple paradigms. We will however verify that the participant did not inadvertently fell asleep during one of the fMRI scans. 
+For accurate estimation of task activation, it is essential to check the quality of both the fMRI images and the task behavior<sup>[1]</sup>. 
+As such, it is important to verify that the subjects were attempting to perform the instructed task (i.e not sleeping and not responding randomly). 
+One particularity of this project is that our single participant is the principal investigator of the project. As such, we trust that the participant, when actively awake, has enough incentive to follow the tasks accurately, even more so that the tasks are simple paradigms. 
```

Comment:
```suggestion
Because the one participant in Cohort I is the principal investigator, we assume that he is following all the instructions and reporting any issues as they occur.
```